resources/scss/app-v2.scss

```diff
@@ -163,3 +163,31 @@
         }
     }
 }
+
+.profileButton {
```

Comment:
I think we can get this done with native Tailwind. Can we try converting it over?

Reply:
If we're going to do that I would rather just make a new button style, didn't we want to do that anyways?

Reply:
Yeah I really want to re-style the button, this is probably a good time to do it.

Reply:
An initial first pass. Do we want to go a different direction w/ our buttons or just keep the same aesthetic with similar styling?
<img width="756" alt="Screen Shot 2021-05-17 at 10 33 19 AM" src="https://user-images.githubusercontent.com/1916366/118506572-6dcb0400-b6fb-11eb-8738-92acfb7941d7.png">

Reply:
I really like this, I think we just need to add some rounded corners to match the rest of the design :).

Reply:
Bam done 
<img width="755" alt="Screen Shot 2021-05-19 at 2 36 16 PM" src="https://user-images.githubusercontent.com/1916366/118866046-9a278180-b8af-11eb-8361-3274a4e0b96e.png">

Reply:
Changed to the karl-orange variable
<img width="779" alt="Screen Shot 2021-05-19 at 2 41 17 PM" src="https://user-images.githubusercontent.com/1916366/118866748-508b6680-b8b0-11eb-9eeb-6fa97942e213.png">